code/__HELPERS/icons.dm

```diff
@@ -915,6 +919,12 @@ var/global/list/friendly_animal_types = list()
 		final_average = BlendRGB(final_average, colour, 1)
 	return final_average
 
+/proc/IsBlankIcon(icon/I)

```

Comment:
How did you test it? This is what I did:
`copy = image(icon=curicon, icon_state=curstate, layer=A.layer, dir=curdir)
if(!copy.icon && !copy.icon_state)
	copy = image(icon=curicon, icon_state=curstate, layer=A.layer, dir=SOUTH)`

and the if statement did not evaluate as true when it should have.